tests/internal/scripts/ccuda/missing_kernel_config.py

```diff
@@ -15,4 +15,4 @@ def opp(a, b):
     #call kernel launch without specifying config
     opp(a, b)
```

Comment:
this test should be done in the same way as the tests in `tests/cuda_test/test_kernel_semantic.py`
you can just move these to `tests/cuda_test/test_kernel_semantic.py` maybe having all the error test in one file will be easier to maintain.

Reply:
Fixed 4f61a29cc5b955aac66a97da402068ae7eeb4e55